github/resource_github_repository.go

```diff
@@ -374,6 +405,18 @@ func resourceGithubRepositoryUpdate(d *schema.ResourceData, meta interface{}) er
 		}
 	}
 
+	if d.HasChange("vulnerability_alerts") {
```

Comment:
my only suggestion here could be to save an API call after creation of a resource by adding the`!d.IsNewResource()` since `Update` immediately follows `Create`